functions/Copy-SqlLogin.ps1

```diff
@@ -282,7 +282,7 @@ Limitations: Does not support Application Roles yet
 								$sid = "0x"; $sourcelogin.sid | % { $sid += ("{0:X}" -f $_).PadLeft(2, "0") }
 								$sqlfailsafe = "CREATE LOGIN [$username] WITH PASSWORD = $hashedpass HASHED, SID = $sid, 
 												DEFAULT_DATABASE = [$defaultdb], CHECK_POLICY = $checkpolicy, 
-												CHECK_EXPIRATION = $checkexpiration, DEFAULT_LANGUAGE = [$language]"
+												CHECK_EXPIRATION = $checkexpiration, DEFAULT_LANGUAGE = [$sourcelogin.Language]"
```

Comment:
$language is only used as a failback when the other two methods don't work. I can't recall the circumstances that cause it, though. Nevertheless, the $sourcelogin.Language is embedded in a string so yo would either have to do $language = $sourcelogin.Language then use $language or $($sourcelogin.Language) within the quoted text.